packages/pwa-kit-react-sdk/src/ssr/server/react-rendering.js

```diff
@@ -208,29 +235,79 @@ export const render = async (req, res, next) => {
     }
 }
```

Comment:
@bendvc, I looked in the react-query guideline, and they have a section about clearing cache on the server, do we need to do that in our case? https://tanstack.com/query/v4/docs/guides/ssr#high-memory-consumption-on-server

Reply:
Thanks for pointing that out. I think if you read the comment carefully you'll see that you actually don't need to call any clear cache methods, as long as you haven't changed the `cacheTime` value. I attached the excerpt below. 

`
On the server, cacheTime defaults to Infinity which disables manual garbage collection and will automatically clear memory once a request has finished. If you are explicitly setting a non-Infinity cacheTime then you will be responsible for clearing the cache early.
`